robot_position_loader/BBox3D[] bbox3d_vec
